src/agni/server/manager/LoginManager.java

```diff
@@ -14,11 +24,51 @@ public LoginManager(InfoSender infoSender,
         this.infoSender = infoSender;
         this.userDataGuard = userDataGuard;
     }
-
+    private byte [] generateSalt() {
+        final Random r = new SecureRandom();
+        byte[] salt = new byte[32];
+        r.nextBytes(salt); 
+        return salt; 
+    }
+    
+    private String generatePasswordHash(String password, byte [] salt) throws InvalidKeySpecException, NoSuchAlgorithmException {
+        KeySpec spec = new PBEKeySpec(password.toCharArray(), salt, 65536, 128);
+        SecretKeyFactory f = SecretKeyFactory.getInstance("pbkdf2withhmacsha1");
+        byte[] hash = f.generateSecret(spec).getEncoded();
+        return new String(hash,StandardCharsets.US_ASCII);
+    }
     @Override
     public void loginRequest(String ip, String user, String password) {
-        // TODO Auto-generated method stub
+        if (password == userDataGuard.getPasswordHash(user)) {
```

Comment:
You need to get the hash for password. Right now this would be  comparing something like 
`gyrosite == 39UGDBH39RHSGDSGHP8T0WGDGDS`